cli/aws_orbit/remote_files/cert.py

```diff
@@ -0,0 +1,71 @@
+""" Module to deploy self signed certs and upload it to IAM """
+import logging
+from aws_orbit import sh
+import boto3
+from botocore.exceptions import ClientError
+import shlex
+import subprocess
+
+_logger: logging.Logger = logging.getLogger(__name__)
+
+def run_command(cmd):
```

Comment:
missing return. this won't pass type checking

Reply:
Fixed